platform/11-tekton-chains.sh

```diff
@@ -31,7 +31,7 @@ esac
 kubectl patch \
```

Comment:
This patch appears to override `artifacts.taskrun.format` even if `TKN_CHAINS_FORMAT=intoto` is set. May need to move the patching of `artifacts.taskrun.format` inside the above case and only patch `artifacts.taskrun.storage` here.

Reply:
Good point, will fix.

Reply:
@bradbeck updated